gateway/src/service.rs

```diff
@@ -202,21 +203,22 @@ impl GatewayService {
         Self { provider, db }
     }
 
-    pub async fn route_fqdn(&self, req: Request<Body>) -> Result<Response<Body>, Error> {
-        let fqdn = req
-            .headers()
-            .typed_get::<Fqdn>()
-            .ok_or_else(|| Error::from(ErrorKind::CustomDomainNotFound))?;
-        let project_name = self.project_name_for_custom_domain(&fqdn).await?;
+    // pub async fn route_fqdn(&self, req: Request<Body>) -> Result<Response<Body>, Error> {
```

Comment:
I assume this was commented out because this calls `self.route` which needs a `ScopedUser`, and that we don't have one here (because this is meant for the proxy, not the control plane of the deployer). This should only be needed in the proxy though.

Reply:
Yes, but I also never used it in the proxy. It actually gets removed in #473